src/processes/sum.json

```diff
@@ -0,0 +1,86 @@
+{
+  "id": "sum",
+  "summary": "Addition of a sequence of numbers",
```

Comment:
ignore_nodata is still mentioned here. Have you tested how sum works in GEE with no data values?

Reply:
I forgot to remove some parts where `ignore_data` is mentioned. GEE just ignores them so it is always true. We have to ask Google if can somehow get a control over nodata values.

Reply:
I doubt we can anytime soon.